js/tv_browser.js

```diff
@@ -1,2 +1,36 @@
 // API Docs at:
-// http://www.tvmaze.com/api
+// // http://www.tvmaze.com/api
+
+
+angular
+.module("tv_show",[])
+.controller("control",ControlFunction)
+
+function ControlFunction (){
+  console.log("Am the controller!")
+  $("#show-select").hide()
```

Comment:
If possible, we tend not to mix jQuery with Angular. This assignment was intended to be done without Angular, but still possible!